electron/utils.ts

```diff
@@ -175,6 +192,12 @@ const launchGame = async (appName: string) => {
   const fixedWinePrefix = winePrefix.replace('~', home)
   let wineCommand = `--wine ${wineVersion.bin}`
 
+  let fixedLauncherArgs = launcherArgs
+
+  if (!(await isOnline()) && !fixedLauncherArgs.includes('--offline')) {
+    fixedLauncherArgs = fixedLauncherArgs.trimRight() + ' --offline'
```

Comment:
Doing like this, the `--offline` flag will be at the end of the arguments, right? 🤔 
So I think it won't work when multiple arguments are passed. 
Reading the Legendary docs, it says that the arguments after the appName are passed to the game, it is not that specific about the --offline flag, but I think, just to be safe, that the flag should be always the first argument when launching the game.

Reply:
otherwise, we could have something like:
for instance: `legendary launch app --arg1 --arg2 --offline and legendary might pass the flag to the game instead of using it to play offline.

Reply:
Fixed!